python/ray/util/rpdb.py

```diff
@@ -170,12 +170,17 @@ def connect_ray_pdb(host=None, port=None, patch_stdstreams=False, quiet=None):
     return rdb
 
 
-def set_trace(host=None, port=None, patch_stdstreams=False, quiet=None):
+def set_trace():
```

Comment:
Can you also add this to `doc/source/package-ref.rst` somewhere so users can search for this? Also, it'd be great to crosslink that API reference in the `ray-debugging.rst`